### ISSUE TITLE
bun install puppeteer causes an error

### ISSUE THREAD

**Issue by dimka3553** [oven-sh/bun#6269]
### What version of Bun is running?

1.0.4

### What platform is your computer?

Darwin 23.0.0 arm64 arm

### What steps can reproduce the bug?

`bun i puppeteer`

and run the following code (`bun index.ts`):

```js
import puppeteer from "puppeteer";

(async () => {
  const browser = await puppeteer.launch();
  const page = await browser.newPage();
  await page.goto("https://example.com");
  await page.screenshot({ path: "example.png" });

  await browser.close();
})();
```

when bun install was is used the following error occurs:
```
error: Could not find Chrome (ver. 117.0.5938.92). This can occur if either
 1. you did not perform an installation before running the script (e.g. `npm install`) or
 2. your cache path is incorrectly configured (which is: /Users/dimashkabura/.cache/puppeteer).
For (2), check out our guide on configuring puppeteer at https://pptr.dev/guides/configuration.
      at resolveExecutablePath 
```

when installed with npm, the code works as expected

### What is the expected behavior?

for the code to work fine

### What do you see instead?

an error

**Comment by Electroid** [comment]
Duplicate of #4959

**Comment by Jarred-Sumner** [comment]
Fixed by @dylan-conway and @paperdave in #7132

This will be part of the Bun v1.0.17 release, which ships in a few hours from the time of writing.

In the meantime, you can run `bun upgrade --canary`
